### PR TITLE
Registration request for teamtanayjobchallenge from RahulChugwani

### PR DESCRIPTION
**Pull request by RahulChugwani**
Hey Maintainer!

I know I am a bit late because I was not feeling well yesterday but I want to contribute and take part in this. 
Kindly accept my request.

### CODE REVIEW

**Comment by gkanishk** [comment]
The **first round** of registrations are closed and hence we are not accepting this **PR** right now. Stay tuned for further updates on [t.me/tanaypratap](t.me/tanaypratap) and apply again when the registration re-opens in a few days. This PR might not be considered for that round or will be considered only at the end. Therefore, it would be better to raise a fresh PR when the registration announcement is made.

**Comment by RahulChugwani** [comment]
Hey Maintainer,
Kindly accept the Pull Request. I want to register for teamtanayjobchallenge. Please accept the request.

File: content/participants/ttjc@RahulChugwani/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: RahulChugwani
+type: "participant"
+name: "Rahul Chugwani"
+portfolio: "https://tanaypratap.com"
```
Comment (gkanishk) [inline content/participants/ttjc@RahulChugwani/index.md]: Kindle change the portfolio link with your portfolio or make it blank if not made yet.

**Comment by RahulChugwani** [comment]
Dear Maintainer,
       I have made the required changes. Please have a look. Kindly accept my PR.

**Pull request merged**